SpatialGDK/Documentation/content/technical-overview/gdk-principles.md

```diff
@@ -18,7 +18,7 @@ The SpatialOS Game Development Kit (GDK) for Unreal is a plugin which allows you
 
     You don’t have to make these technical tradeoffs with the GDK. SpatialOS can spread computation across multiple servers, allowing for far more complex games and much higher player counts.</br></br>
 
-    > This will be enabled by [Zoning]({{urlRoot}}/content/technical-overview/gdk-concepts#zoning), which is currently in pre-alpha. We invite you to try out the [Multiserver Shooter tutorial]({{urlRoot}}/content/tutorials/multiserver-shooter/tutorial-multiserver-intro) and learn about how it works, but we don’t recommend you start developing features that use zoning yet.
+    > This will be enabled by [zoning]({{urlRoot}}/content/technical-overview/gdk-concepts#zoning), which is currently in pre-alpha. We invite you to try out the [multiserver zoning shooter tutorial]({{urlRoot}}/content/tutorials/multiserver-shooter/tutorial-multiserver-intro) and learn about how it works, but we don’t recommend you start developing features that use zoning yet.
 
```

Comment:
Here shouldn't we add that it's either zoning or offloading?

Reply:
Hmm, I'm not sure - maybe we need to check with Ernie/an engineer about whether we want to mention both here, or whether we want to talk about zoning specifically in this case.

Reply:
Yes, it's both. Both offer "spread computation across multiple servers, allowing for far more complex games and much higher player counts" with Offloading being the one to promote, zoning the one to downplay as it's in pre-alpha.
Making a suggestion.